packages/apps/storybook/package.json

```diff
@@ -5,13 +5,15 @@
   "devDependencies": {
     "@babel/core": "^7.12.10",
     "@bentley/bentleyjs-core": "2.11.0",
+    "@storybook/builder-webpack5": "6.4.14",
+    "@storybook/manager-webpack5": "6.4.14",
     "@bentley/frontend-authorization-client": "2.11.0",
     "@bentley/itwin-client": "2.11.0",
     "@itwin/create-imodel-react": "^1.0.0",
     "@itwin/delete-imodel-react": "^1.0.0",
     "@itwin/delete-itwin-react": "^1.0.0",
     "@itwin/imodel-browser-react": "~1.1.1",
-    "@itwin/itwinui-react": "^2.11.4",
+    "@itwin/itwinui-react": "^3.0.4",
     "@itwin/manage-versions-react": "~1.2.0",
```

Comment:
Yeah it should be but it is how its always been I guess

Reply:
should be fixed

Reply:
this should be a simple change why cant we squeeze this into this pr which has already grown quite out of scope already?

Reply:
@aruniverse I'm trying it but it doesn't seem to be simple since it requires full update `rush update --full` which updates all the packages and broke this repo on so many points